composer.json

```diff
@@ -13,14 +13,17 @@
         {
             "name": "Patrick Brouwers",
             "email": "patrick@maatwebsite.nl"
+        },
+        {
+            "name": "Tom H Anderson",
+            "email": "tom.h.anderson@gmail.com"
         }
     ],
     "require": {
-        "php": "^8.0",
-        "doctrine/annotations": "^2",
-        "doctrine/dbal": "^3.2",
-        "doctrine/orm": "^2.14",
-        "doctrine/persistence": "^3",
+        "php": "^8.1",
+        "doctrine/dbal": "^4.1",
+        "doctrine/orm": "^3.1",
```

Comment:
Not sure how this is handled on this repository usually, but it might be smart to create releases that support ORM 2 and 3 first, so people can migrate one package at a time. Same for DBAL.

Reply:
ORM 3 requires DBAL 4.  What my PR proposes is a major version release without a path to migrate "gracefully".

Reply:
> It's like the whole team (10 of you?) has dropped their assumed goal of maintaining a world-class library.

Note that Doctrine is not related to this library. I have been the sole "maintainer" of this library for several years. But I almost never contribute myself, it's up to the community to provide pull requests, unless none have provided a pull request before laravel goes out of security updates.

Yes, laravel-doctrine needs a proper maintainer which can lift this package up to latest standard and keep constant progress.

Reply:
But this isn't you:
https://github.com/laravel-doctrine/orm/blob/2.0/LICENSE#L3
And this isn't you:
https://github.com/laravel-doctrine/orm/blob/2.0/composer.json#L14-L15

I would like to be that proper maintainer for this organization.  I have many Doctrine ORM and Laravel libraries at https://github.com/api-skeletons  I am also a member of the Doctrine open source team.  My latest project, the LDOG Stack, https://github.com/api-skeletons/ldog, relies on this library as well as my other projects.  Reviving this resource is in my best interest.

To see my latest work please review https://github.com/api-skeletons/doctrine-orm-graphql (and be sure to see the documentation).

Reply:
I dont feel that I have contributed enough to this repo that I feel I should be on those lists.. :) 

I will take a look at your profile. I don't think it will be any problem of giving you access 👌

Reply:
> ORM 3 requires DBAL 4.

It does not.

Reply:
As you can see in this composer.json, libraries can be compatible with both ORM 2 and 3 and DBAL 3 and 4.
https://github.com/API-Skeletons/doctrine-orm-graphql/blob/12.2.x/composer.json#L14-L22

I don't foresee version 3 of this library trying to do the same.  So I think to catch this library up, ORM 3 should be supported at the highest versions.